hw/ip/entropy_src/dv/cov/entropy_src_UNR.vRefine

```diff
@@ -24,10 +24,12 @@
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/32" entityType="block" excTime="1665673903" file="0" im-checksum="786255092" line="111" name="exclude" reviewer="unknown" text="assign alert_test                   = Stub ? stub_alert_test    : core_alert_test;" user="1" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/35" entityType="block" excTime="1665673907" file="0" im-checksum="786255092" line="112" name="exclude" reviewer="unknown" text="assign alert                        = Stub ? stub_alert         : core_alert;" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/2" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/1" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/0" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_sink&quot;/0" entityType="toggle-bit" excTime="1665674022" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_i.a_param&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_i.a_user.rsvd&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_sink&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;alert_rx_i[0].ping_p&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;alert_rx_i[0].ping_n&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
```

Comment:
As mentioned in a comment in the other PR, these should have already been excluded by our common Xcelium coverage exclusion tcl script:
https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/xcelium/common_cov_excl.tcl#L6

It is worth investigating why it is not taking effect.